examples/compiler/01_iet.ipynb

```diff
@@ -0,0 +1,696 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial describes one particular intermediate representation used by the Devito Compiler: the Iteration/Expression Tree (IET), a special type of Abstract Syntax Tree (AST).\n",
+    "\n",
+    "# Part I - Top Down\n",
+    "\n",
+    "First, let's describe a $Domain$, and a $Function$ that will allow us to specify how such a domain gets modified. In particular, we will look at Functions that change through $time$. \n",
```

Comment:
I suggest to use:

- \` ... \` for Devito objects, such as Function and Operator, so if you write \`Function\`  it gets rendered as `Function`
- \_ ... \_ for relevant names, such as "domain". This gets rendered to italics

Use dollars only for mathematical symbols, which should never be the case in these tutorials.

I'm writing this here, but it should be applied to both notebooks